presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -101,10 +103,15 @@ public void testShowCreateView()
     {
         onHive().executeQuery("DROP VIEW IF EXISTS hive_show_view");
 
-        onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT * FROM nation");
+        onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT n_nationkey FROM nation");
 
-        // view SQL depends on Hive distribution
-        assertThat(query("SHOW CREATE VIEW hive_show_view")).hasRowsCount(1);
+        QueryResult result = query("SHOW CREATE VIEW hive_show_view");
+
+        assertThat(result).hasRowsCount(1);
+        String actualSql = (String) result.row(0).get(0);
+        assertEquals(
+                actualSql.replaceAll("\n", " ").replaceAll("\\s+", " "),
+                "CREATE VIEW hive.default.hive_show_view AS SELECT \"n_nationkey\" FROM \"default\".\"nation\"");
```

Comment:
we need to use something else that `hive` catalog name here.

Reply:
In product tests we have few hive catalogs mapped to same metastore.
So you can access the Hive view from two of them within single test.